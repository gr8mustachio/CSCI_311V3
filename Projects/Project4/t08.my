Adjacency lists of the graph are:
9 
2 4 5 7 
1 3 
2 4 
1 3 
1 6 
5 9 7 
6 8 1 
7 9 
0 6 8 
Are there two distinct paths between nodes 6 and 8?
hello
There are NO two distinct paths between nodes 6 and 8.
Are there two distinct paths between nodes 3 and 9?
hello
There are NO two distinct paths between nodes 3 and 9.
