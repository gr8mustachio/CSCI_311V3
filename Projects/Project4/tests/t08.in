10
addEdge 0 9
addEdge 1 2
addEdge 4 1
addEdge 3 2
addEdge 3 4
addEdge 1 5
addEdge 5 6
addEdge 9 6
addEdge 7 6
addEdge 7 8
addEdge 8 9
addEdge 1 7
printGraph
distinctPaths 6 8
distinctPaths 3 9
