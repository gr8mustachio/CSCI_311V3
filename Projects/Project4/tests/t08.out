Adjacency lists of the graph are:
9 
2 4 5 7 
1 3 
2 4 
1 3 
1 6 
5 9 7 
6 8 1 
7 9 
0 6 8 
Are there two distinct paths between nodes 6 and 8?
6 9 8 
6 7 8 
There are two distinct paths between nodes 6 and 8.
Are there two distinct paths between nodes 3 and 9?
3 2 1 5 6 9 
3 4 1 7 8 9 
There are two distinct paths between nodes 3 and 9.
